SparseGrids/tasgridWrapper.cpp

```diff
@@ -424,10 +463,10 @@ void TasgridWrapper::outputHierarchicalCoefficients() const{
     int num_p = grid->getNumPoints();
     int num_d = grid->getNumOutputs();
     if (outfilename != 0){
-        writeMatrix(outfilename, num_p, num_d, coeff, useASCII);
+        writeMatrix(outfilename, num_p, num_d, coeff, useASCII, grid->isFourier());
```

Comment:
Why not handle `printMatrix` the same as `writeMatrix`?
```
printMatrix(num_p, (grid->isFourier() ? 2 * num_d : num_d), coeff);
```

Reply:
Because `printMatrix()` is human readable and the format should be
```
<number> + <number> i  <number> + <number> i ...
```
I must have missed to add the comment regarding `printMatrix()`